### PR TITLE
BD-3035 Add permission sets and roles docs

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-3035**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/administrative/app_settings/manage_your_braze_users/user_permissions.md [hunk 1]
```diff
@@ -114,6 +112,29 @@ Marketers | “Access Campaigns, Canvases, Cards, Feature Flags, Segments, Media
...
+
+| Role Name    | Workspace | Permissions  
+----------- | ----------- | ---------
+| Marketer - Fashion Brands | {::nomarkdown}[DEV] Fashion Brand, [QA] Fashion Brand, [PROD] Fashion Brand {:/} | “Access Campaigns, Canvases, Cards, Feature Flags, Segments, Media Library, and Preference Center"<br>“Manage Media Library Assets” |
+| Marketer - Skincare Brands | {::nomarkdown}[DEV] Skincare Brand, [QA] Skincare Brand, [PROD] Skincare Brand {:/} | “Access Campaigns, Canvases, Cards, Feature Flags, Segments, Media Library, and Preference Centers” <br>“Manage Media Library Assets” |
+| User Management - All Brands | {::nomarkdown}[DEV] Fashion Brand, [QA] Fashion Brand, [PROD] Fashion Brand, [DEV] Skincare Brand, [QA] Skincare Brand, [PROD] Skincare Brand {:/} | “Manage Dashboard Users”<br>“Manage Teams” |
+{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
+
+### Assigning roles
```
Comment (rachel-feinberg) [inline _docs/_user_guide/administrative/app_settings/manage_your_braze_users/user_permissions.md:134]:
```suggestion
#### Assigning roles
```

File: _docs/_user_guide/administrative/app_settings/manage_your_braze_users/user_permissions.md [hunk 2]
```diff
@@ -114,6 +112,29 @@ Marketers | “Access Campaigns, Canvases, Cards, Feature Flags, Segments, Media
 User Management | “Manage Dashboard Users” <br> “Manage Teams”
 {: .reset-td-br-1 .reset-td-br-2}
 
+### Assigning permission sets
```
Comment (rachel-feinberg) [inline _docs/_user_guide/administrative/app_settings/manage_your_braze_users/user_permissions.md:115]:
```suggestion
#### Assigning permission sets
```

File: _docs/_user_guide/administrative/app_settings/manage_your_braze_users/user_permissions.md [hunk 3]
```diff
@@ -97,15 +97,13 @@ Individual users can be granted different degrees of access on an app-by-app bas
...
-On the **Permission Sets** page found at **Settings** > **Permission Settings**, you can create, edit, and delete custom permission assignments for your Braze dashboard users. 
+You can bundle together relevant permissions with permission sets and roles by navigating to **Settings** > **Permission Settings**.
 
-Permission sets can be assigned by selecting an existing user or creating a new one on the **Company Users** page, then choosing the appropriate permission set under **Workspace Level Permissions**.
+### Permission sets
 
-![The Permission Sets tab opens a Manage Permissions Sets page with a table of permission set names and selected permissions.][5]
-
-Each permission set can be defined for a specific group of users, such as in the following example:
+Use permission sets to bundle together permissions that are related to specific subject areas or actions. They can be applied to dashboard users who need the same access across different workspaces, such as in the following example:
```
Comment (lydia-xie) [inline _docs/_user_guide/administrative/app_settings/manage_your_braze_users/user_permissions.md:106]:
```suggestion
Use permission sets to bundle permissions related to specific subject areas or actions. They can be applied to dashboard users who need the same access across different workspaces, such as in the following example:
```

File: _docs/_user_guide/administrative/app_settings/manage_your_braze_users/user_permissions.md [hunk 4]
```diff
@@ -97,15 +97,13 @@ Individual users can be granted different degrees of access on an app-by-app bas
 |App Level Settings|Determines the user's level of editing access.|
 {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
 
-## Permission sets
+## Permission sets and roles
 
-On the **Permission Sets** page found at **Settings** > **Permission Settings**, you can create, edit, and delete custom permission assignments for your Braze dashboard users. 
+You can bundle together relevant permissions with permission sets and roles by navigating to **Settings** > **Permission Settings**.
```
Comment (lydia-xie) [inline _docs/_user_guide/administrative/app_settings/manage_your_braze_users/user_permissions.md:102]:
```suggestion
You can bundle relevant permissions with permission sets and roles by navigating to **Settings** > **Permission Settings**.
```

**Review by lydia-xie — APPROVED** [review]
lgtm! made 2 small edits

**Pull request merged**